replicated-security/duality-testnet-1/proposal-start-duality-testnet-1.json

```diff
@@ -0,0 +1,19 @@
+{
+    "title": "Create the duality chain",
+    "description": "# Launching the duality-testnet-1 Chain\r\n\r\nThis is the proposal to create the first persistent duality chain, \"duality-testnet-1\".\r\n\r\nPlease visit the [cosmos/testnets](https://github.com/cosmos/testnets/tree/master/replicated-security/duality-testnet-1/) repo for additional details.",
+    "chain_id": "duality-testnet-1",
```

Comment:
This doesn't match the genesis file chain ID

Reply:
Done.